qa/suites/rados/cephadm/mon_election

```diff
@@ -0,0 +1 @@
+.qa/mon_election
```

Comment:
This needs to go in the cephadm subdirs
```
IndexError: list index out of range
```
from e.g., /a/sage-2020-02-23_16:41:52-rados-wip-sage2-testing-2020-02-23-0755-distro-basic-smithi/4793867

Reply:
What? mon_election is a symlink and the only cephadm subduers are eg "tasks". Those extra parameters go at this level everywhere else and it's next to the msgr/ and fixed-2.yaml links.

There's definitely something going wrong since the job managed to get a job with description "rados/cephadm/mon_election/classic.yaml" and an empty tasks list, but I don't see how this did it. :/

Reply:
you may need to rebase?  on master,
```
gnit:build (fix-44121) 10:49 AM $ ls -al ../qa/suites/rados/cephadm/
total 4
drwxrwxr-x   6 sage sage   79 Feb 23 18:05 ./
drwxrwxr-x. 25 sage sage 4096 Feb 19 08:06 ../
lrwxrwxrwx   1 sage sage    6 Feb 19 08:06 .qa -> ../.qa/
drwxrwxr-x   3 sage sage   78 Feb 24 10:41 smoke/
drwxrwxr-x   2 sage sage  128 Feb 24 10:41 upgrade/
drwxrwxr-x   5 sage sage  115 Feb 19 08:06 with-work/
drwxrwxr-x   4 sage sage   52 Feb 19 08:06 workunits/
gnit:build (fix-44121) 01:19 PM $ ls -al ../qa/suites/rados/cephadm/*
../qa/suites/rados/cephadm/smoke:
total 8
-rw-rw-r-- 1 sage sage   0 Feb 19 08:06 %
drwxrwxr-x 3 sage sage  78 Feb 24 10:41 ./
drwxrwxr-x 6 sage sage  79 Feb 23 18:05 ../
drwxrwxr-x 2 sage sage 260 Feb 23 18:05 distro/
-rw-rw-r-- 1 sage sage 299 Feb 23 18:05 fixed-2.yaml
lrwxrwxrwx 1 sage sage   6 Feb 19 08:06 .qa -> ../.qa/
-rw-rw-r-- 1 sage sage 230 Feb 24 10:41 start.yaml

../qa/suites/rados/cephadm/upgrade:
total 12
-rw-rw-r-- 1 sage sage   0 Feb 23 18:05  %
drwxrwxr-x 2 sage sage 128 Feb 24 10:41  ./
drwxrwxr-x 6 sage sage  79 Feb 23 18:05  ../
-rw-rw-r-- 1 sage sage 136 Feb 24 10:41  1-start.yaml
-rw-rw-r-- 1 sage sage 113 Feb 23 18:05  2-start-upgrade.yaml
-rw-rw-r-- 1 sage sage 328 Feb 23 18:05  3-wait.yaml
lrwxrwxrwx 1 sage sage  15 Feb 23 18:05 'distro$' -> ../smoke/distro/
lrwxrwxrwx 1 sage sage  21 Feb 23 18:05  fixed-2.yaml -> ../smoke/fixed-2.yaml
lrwxrwxrwx 1 sage sage   6 Feb 23 18:05  .qa -> ../.qa/

../qa/suites/rados/cephadm/with-work:
total 4
-rw-rw-r-- 1 sage sage   0 Feb 19 08:06 %
drwxrwxr-x 5 sage sage 115 Feb 19 08:06 ./
drwxrwxr-x 6 sage sage  79 Feb 23 18:05 ../
drwxrwxr-x 2 sage sage 194 Feb 19 08:06 distro/
lrwxrwxrwx 1 sage sage  25 Feb 19 08:06 fixed-2.yaml -> .qa/clusters/fixed-2.yaml
drwxrwxr-x 2 sage sage  55 Feb 19 08:06 mode/
lrwxrwxrwx 1 sage sage   8 Feb 19 08:06 msgr -> .qa/msgr/
lrwxrwxrwx 1 sage sage   7 Feb 19 08:06 .qa -> ../.qa//
-rw-rw-r-- 1 sage sage  92 Feb 19 08:06 start.yaml
drwxrwxr-x 2 sage sage  70 Feb 19 08:06 tasks/

../qa/suites/rados/cephadm/workunits:
total 0
-rw-rw-r-- 1 sage sage   0 Feb 19 08:06 %
drwxrwxr-x 4 sage sage  52 Feb 19 08:06 ./
drwxrwxr-x 6 sage sage  79 Feb 23 18:05 ../
drwxrwxr-x 2 sage sage 101 Feb 19 08:06 distro/
lrwxrwxrwx 1 sage sage   6 Feb 19 08:06 .qa -> ../.qa/
drwxrwxr-x 2 sage sage  88 Feb 24 10:41 task/
```

Reply:
Oh yep, it's happened to me again. *cries*